sprint6/zadanie4.js

```diff
@@ -0,0 +1,22 @@
+/*
+  Zadanie 4. Napisz funkcję, która przyjmie liczbę całkowitą i wypisze wszystkie liczby
+  od tej liczby całkowitej do 100. Jeżeli użytkownik poda liczbę większą niż 100, poinformuj
+  go żeby wpisał mniejszą liczbę niż 100
+*/
+
+function writeNumber(n) {
+  if (n <= 100) {
+    for (let i = 60; i <= n; i++){
```

Comment:
W tym momencie nieważne jaką liczbę przekażemy, nasza pętla zawsze będzie wypisywała liczby od 60. To jest niezgodne z poleceniem zadania.